share/spack/gitlab/cloud_pipelines/stacks/radiuss/spack.yaml

```diff
@@ -80,18 +78,18 @@ spack:
           image: { "name": "ghcr.io/scottwittenburg/ecpe4s-ubuntu18.04-runner-x86_64:2020-09-01", "entrypoint": [""] }
           tags: ["spack", "public", "large", "x86_64"]
     temporary-storage-url-prefix: "s3://spack-binaries-prs/pipeline-storage"
+    broken-specs-url: "s3://spack-binaries-prs/broken-specs"
     service-job-attributes:
       image: { "name": "ghcr.io/scottwittenburg/ecpe4s-ubuntu18.04-runner-x86_64:2020-09-01", "entrypoint": [""] }
       before_script:
         - . "./share/spack/setup-env.sh"
         - spack --version
-        #- cd ${SPACK_CONCRETE_ENV_DIR}   # TODO: Once rebase to #22887
-        - cd share/spack/gitlab/cloud_pipelines/stacks/radiuss
+        - cd ${SPACK_CONCRETE_ENV_DIR}
```

Comment:
I just checked on the service jobs we generate, and currently we're not providing them with the concrete environment from the pipeline generation job.  So even though that variable will be defined and appear in the generated job variables, the directory will be empty, and activating an environment from there won't work.  We probably should pass the concrete environment to the service jobs, but until that's fixed, can you change this line back to the way you had it?

```
cd share/spack/gitlab/cloud_pipelines/stacks/radiuss
```